tests/run_pretrained_models.py

```diff
@@ -436,6 +437,7 @@ def main():
                 t = tests[test]
                 if t.perf:
                     f.write("{},{},{}\n".format(test, t.tf_runtime, t.onnx_runtime))
+    return failed
```

Comment:
where is it defined?

Reply:
what do you mean by "it"?
if "it" is "failed", then it's defined the same function.